From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Fri, 19 Nov 2021 15:24:32 +0100
Subject: [PATCH] Release 5.16/customers/ptx/litex/20211119-2

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9e12c14ea0fb..870a8a87b013 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 16
 SUBLEVEL = 0
-EXTRAVERSION = -rc1
+EXTRAVERSION = -rc1-20211119-2
 NAME = Trick or Treat
 
 # *DOCUMENTATION*
