From a96ca4706cc13243e76e77775195283c5e88dbdb Mon Sep 17 00:00:00 2001
From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Fri, 23 Jul 2021 11:09:33 +0200
Subject: [PATCH] soc/software: bios.elf link statically

Signed-off-by: Steffen Trumtrar <s.trumtrar@pengutronix.de>
---
 litex/soc/software/bios/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/litex/soc/software/bios/Makefile b/litex/soc/software/bios/Makefile
index 6c0a6aac..d2481cbf 100755
--- a/litex/soc/software/bios/Makefile
+++ b/litex/soc/software/bios/Makefile
@@ -63,6 +63,7 @@ vpath %.a $(PACKAGES:%=../%)
 		$(PACKAGES:%=-L../%) \
 		-Wl,--whole-archive \
 		-Wl,--gc-sections \
+		-Wl,--no-dynamic-linker \
 		$(LIBS:lib%=-l%) \
 
 
-- 
2.32.0

